webapp/src/component/layout/Notifications/MfaDisabledItem.tsx

```diff
@@ -4,18 +4,18 @@ import {
   NotificationItem,
   NotificationItemProps,
 } from 'tg.component/layout/Notifications/NotificationItem';
+import { LINKS } from 'tg.constants/links';
 
 type MfaDisabledItemProps = NotificationItemProps;
 
 export const MfaDisabledItem: FunctionComponent<MfaDisabledItemProps> = ({
   notification,
   ...props
 }) => {
-  const destinationUrl = '/account/security';
   return (
     <NotificationItem
       notification={notification}
-      destinationUrl={destinationUrl}
+      destinationUrl={LINKS.USER_ACCOUNT_SECURITY}
```

Comment:
It's better to pass just result string like this:
```
LINKS.USER_ACCOUNT_SECURITY.build()
```

You are making NotificationItem unnecessary complicated